### PR TITLE
WIP: add modal and refactor

### PR DESCRIPTION
**Pull request by RenuSuresh**
* Added React Modal and content of it.
* handled if no data found in search
* refactor Table and LyricsModal

**before:**
image

After:
image

Handle Error if no data available:
image

In Progress:
- UT
- Refactoring code

### CODE REVIEW

File: src/Components/LyricsModal.js [hunk 1]
```diff
@@ -1,41 +1,14 @@
-import React, { useEffect, useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import "./LyricsModal.css";
 import "bootstrap";
 import "popper.js";
 import { getLyrics } from "../axios";
 import logo from "./../assets/Dog-swimming.png";
 import BeatLoader from "react-spinners/BeatLoader";
-import Modal from "react-modal";
+import { default as ReactModal } from "react-modal";
```
Comment (ArfatSalman) [inline src/Components/LyricsModal.js:8]: Why not just do 
` import ReactModal from "react-modal"`

File: src/Components/LyricsModal.js [hunk 2]
```diff
@@ -48,24 +21,51 @@ const customStyles = {
     width: "820px",
   },
 };
+function Modal({ children }) {
+  const value = useContext(ModalContext);
```
Comment (ArfatSalman) [inline src/Components/LyricsModal.js:25]: Which `value`? What is it?

File: src/Components/LyricsModal.js [hunk 3]
```diff
@@ -48,24 +21,51 @@ const customStyles = {
...
+  console.log(typeof value);
```
Comment (ArfatSalman) [inline src/Components/LyricsModal.js:26]: Why is this committed?

**Review by ArfatSalman — COMMENTED** [review]
You can merge after looking at the changes.

File: src/Components/LyricsModal.js (hunk 1 above)
Comment (RenuSuresh) [reply src/Components/LyricsModal.js:8]: yes, changed as per suggestion

File: src/Components/LyricsModal.js (hunk 3 above)
Comment (RenuSuresh) [reply src/Components/LyricsModal.js:26]: will remove.

File: src/Components/LyricsModal.js (hunk 2 above)
Comment (RenuSuresh) [reply src/Components/LyricsModal.js:25]: will give meaningful variable name.

**Pull request merged**